src/test/java/com/marklogic/kafka/connect/sink/ConvertSinkRecordTest.java

```diff
@@ -77,6 +77,24 @@ public void binaryContent() {
 		assertEquals("hello world".getBytes().length, content.get().length);
 	}
 
+	@Test
+	public void emptyContent() {
```

Comment:
I recommend moving these tests to be against MarkLogicSinkTask, which is where I think the only null check is needed. Today, we know that if we pass in a SinkRecord with a null value (or a null SinkRecord), we'll get an error. So the test would simulate that and just verify that no error occurs (we can't easily verify that a log message is written).

Reply:
I have made the recommended changes.